setup.py

```diff
@@ -31,4 +32,7 @@ def get_version(package):
             'lib/linux/*',
         ],
     },
+    install_requires=[
+        'telegram-text>=0.1.1',
```

Comment:
Let's choose a compatible release version, such as `~=0.1`. This way, if there are any incompatible changes in the `1.0` release of `telegram-text`, it won't be installed.